.github/workflows/publish-application-packages.yml

```diff
@@ -0,0 +1,43 @@
+name: Zarf Injector Rust Binaries
+
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+    inputs:
+      branchName:
+        description: "Branch to build the injector from"
+        required: true
+
+jobs:
+  build-injector:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout Repo"
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          ref: ${{ github.event.inputs.branchName }}
+
+      - name: Setup golang
+        uses: ./.github/actions/golang
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/node
+
+      - name: Build CLI
+        run: |
+          make build-cli-linux-amd
```

Comment:
I wonder if we want to do this or use a release version of Zarf to publish packages (since the version will be in the build info)